<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>550</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #1e1e1e;
}</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="mainFrame">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: #0F0F0F;
	border-radius: 5px;
}
QPushButton {
	color: rgb(185, 185, 185);
	border-radius: 5px;
	
	background-color: #232D3F;
}

QPushButton:hover {
	background-color: rgb(71, 91, 127);
}

QComboBox {
    background-color: #2D2D30;
    color: #F1F1F1;
    padding: 8px;
    border-radius: 5px;
}

QComboBox:hover {
    border: 1px solid #232D3F;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox QAbstractItemView {
    color: #F1F1F1;
    background-color: #2D2D30;
    selection-background-color: #3daee9;
}

QLineEdit {
    /* Base styling */
    background-color: #DBF5FF; 
    border: 1px solid #B2E0EA;
    border-radius: 5px;
    padding: 8px 12px;
    color: #333;
    selection-background-color: #AAD5FF;
    border-color: #85C1E9;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="welcomeLabel">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>Welcom to Game Inventory</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gameCombo">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="placeholderText">
         <string>Choose Game</string>
        </property>
        <item>
         <property name="text">
          <string>Pyramid XO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5x5 XO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Connect4</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="modeCombo">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="placeholderText">
         <string>Choose Mode</string>
        </property>
        <item>
         <property name="text">
          <string>PvP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vs Computer</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="player1Line">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Player Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vsIcon">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resource/resource.qrc">:/icons/versus.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="player2Line">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Player Name</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="goButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
